### PR TITLE
[Refactor](scan) refactor scan scheduler to improve performance

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

File: gensrc/thrift/PaloInternalService.thrift [hunk 1]
```diff
@@ -257,6 +257,8 @@ struct TQueryOptions {
   90: optional bool skip_missing_version = false;
 
   91: optional bool runtime_filter_wait_infinitely = false;
+
+  92: optional i32 wait_full_block_schedule_time = 1;
```
Comment (yiguolei) [inline gensrc/thrift/PaloInternalService.thrift:261]: not use time.
using wait_full_block_schedule_multiplier

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Review by kaka11chen — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**